include/libtorrent/create_torrent.hpp

```diff
@@ -347,7 +347,7 @@ namespace libtorrent
 		// and they have the same name.
 		bool m_multifile:1;
 
-		// this is true if the torrent is private. i.e., is should not
+		// this is true if the torrent is private. i.e., its interested peers should not
```

Comment:
The comment for `set_priv()` does not mention DHT.
I propose to add DHT to the comment: 

Torrents with the private flag set ask the client to not use any other
sources than the tracker for peers, and to not **use DHT to** advertise itself publicly,
**only the** tracker.

I changed "...ask clients..." to "...ask the client..." because "itself" is singular.